packages/application-shell/src/components/application-shell/application-shell.js

```diff
@@ -113,217 +114,222 @@ export const RestrictedApplication = props => (
                   appEnv={props.environment.env}
                   defaultFlags={props.defaultFeatureFlags}
                 >
-                  <React.Fragment>
-                    {/* NOTE: the requests in flight loader will render a loading
+                  <VersionTracker>
```

Comment:
Just intentation.

Reply:
Curious as to why it renders children? Why not just `<VersionTracker />`?